index.html

```diff
@@ -26,28 +26,29 @@ <h1>
          <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
       </ul>
       <ul>
-        <img src="shoe2.png" alt="picture of blue and orange sneaker">
-        <img src="shoe3.png" alt="picture of red and black sneaker">
-        <img src="shoe4.png" alt="picture of blue and white sneaker">
+        <img src="shoe2.png" alt="picture of blue and orange sneaker" class="shoe2">
+        <img src="shoe3.png" alt="picture of red and black sneaker" class="shoe3">
+        <img src="shoe4.png" alt="picture of blue and white sneaker" class="shoe4">
     </ul>
     </header>
 
 
-    <img src="shoe1.png" alt="picture of black and white sneaker">
-
+    <img src="shoe1.png" alt="picture of black and white sneaker" class="shoe1">
```

Comment:
You already made rules for each of these "shoe" classes in your `styles.css` file, but you forgot to add them to the images! Adding these lets your styles apply to the shoes.